asrtoolkit/align_json.py

```diff
@@ -3,18 +3,24 @@
 Forced alignment tools CLI
 """
 
-import argparse
+from fire import Fire
 
 from asrtoolkit.alignment import preprocess_gk_json, preprocess_txt
 from asrtoolkit.alignment.align import align
 from asrtoolkit.data_structures.time_aligned_text import time_aligned_text
-from asrtoolkit.file_utils.name_cleaners import basename, sanitize, strip_extension
+from asrtoolkit.file_utils.name_cleaners import (
+    basename,
+    sanitize,
+    strip_extension,
+)
 
 
 def align_json(ref_txt, json_file, filename=None):
     """
-    Wraps alignment tools to allow for forced alignment of a gk json hypothesis against a reference text file
-    Produces a time-aligned stm file named filename using the correct transcript where alignments were detected
+    CLI for forced alignment tools
+    Using a reference txt file and a hypothesis gk json 
+        file, this time-aligns the reference txt file 
+        and outputs an STM file
```

Comment:
I think the docstring should describe the input arguments since this is the help text for the function?

Reply:
Fire documents that stuff for us! Yay. No need to be in the docstring.